<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginScreen</class>
 <widget class="QMainWindow" name="LoginScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>666</width>
    <height>574</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>IsfahanSIS</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="5" column="1">
     <spacer name="mainBottomSpacer">
      <property name="orientation">
       <enum>Qt::Orientation::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>120</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="1">
     <widget class="QGroupBox" name="loginLayout">
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="title">
       <string>Login</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="usernameLabel">
         <property name="text">
          <string>User name:</string>
         </property>
         <property name="buddy">
          <cstring>userNameEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="passwordLabel">
         <property name="text">
          <string>Password:</string>
         </property>
         <property name="buddy">
          <cstring>passwordEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="passwordEdit">
         <property name="font">
          <font>
           <bold>false</bold>
          </font>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::EchoMode::Password</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="2">
        <widget class="QLineEdit" name="userNameEdit">
         <property name="font">
          <font>
           <bold>false</bold>
          </font>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QToolButton" name="showPasswordButton">
         <property name="text">
          <string>Show Password</string>
         </property>
         <property name="icon">
          <iconset resource="../../resources/main.qrc">
           <normaloff>:/root/dialogs/password-show.png</normaloff>:/root/dialogs/password-show.png</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="1">
     <spacer name="mainTopSpacer">
      <property name="orientation">
       <enum>Qt::Orientation::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>121</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="2" rowspan="2">
     <spacer name="mainRightSpacer">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>134</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="0" rowspan="2">
     <spacer name="mainLeftSpacer">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>134</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="4" column="1">
     <layout class="QGridLayout" name="controlLayout">
      <item row="0" column="0">
       <spacer name="controlTopLeftSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="loginButton">
        <property name="font">
         <font>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Login</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <spacer name="controlTopRightSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0">
       <spacer name="controlBottomLeftSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="createAccountButton">
        <property name="font">
         <font>
          <bold>true</bold>
          <underline>true</underline>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(85, 0, 255);</string>
        </property>
        <property name="text">
         <string>Not already have an account?</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <spacer name="controlBottomRightSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="title">
      <property name="font">
       <font>
        <family>Segoe Print</family>
        <pointsize>13</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Welcome to Isfahan Student Information System</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>666</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>userNameEdit</tabstop>
  <tabstop>passwordEdit</tabstop>
  <tabstop>showPasswordButton</tabstop>
  <tabstop>loginButton</tabstop>
  <tabstop>createAccountButton</tabstop>
 </tabstops>
 <resources>
  <include location="../../resources/main.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>showPasswordButton</sender>
   <signal>clicked(bool)</signal>
   <receiver>LoginScreen</receiver>
   <slot>togglePasswordShow(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>577</x>
     <y>299</y>
    </hint>
    <hint type="destinationlabel">
     <x>603</x>
     <y>366</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>createAccountButton</sender>
   <signal>clicked()</signal>
   <receiver>LoginScreen</receiver>
   <slot>openAccountDialog()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>333</x>
     <y>380</y>
    </hint>
    <hint type="destinationlabel">
     <x>78</x>
     <y>449</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>loginButton</sender>
   <signal>clicked()</signal>
   <receiver>LoginScreen</receiver>
   <slot>authorize()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>374</x>
     <y>352</y>
    </hint>
    <hint type="destinationlabel">
     <x>533</x>
     <y>460</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>togglePasswordShow(bool)</slot>
  <slot>openAccountDialog()</slot>
  <slot>authorize()</slot>
 </slots>
</ui>
